deps/spidershim/test/value.cc

```diff
@@ -921,6 +947,361 @@ TEST(SpiderShim, ExternalStringResourceDestructorCalled) {
   EXPECT_TRUE(externalOneByteStringResourceDestructorCalled);
 }
 
+// Begin code copied from THREADED_TEST(StringWrite) in test-api.cc.
```

Comment:
I'm unsure. I suggest it because test-api.cc is copyright 2012. Either way, the notice at the top of value.cc won't be an complete copyright header, just a pointer to other files. So maybe it doesn't matter.